### ISSUE TITLE
Cannot read property 'innerText' of undefined

### ISSUE THREAD

**Issue by Overhaile** [jakiestfu/Medium.js#130]
medium.js:1725 Uncaught TypeError: Cannot read property 'innerText' of undefined

Heres my init:

	var ed = $("#createBox");
	new Medium({
		element: ed[0],
		modifier: 'auto',
		placeholder: "What's happening?",
		autofocus: true
)}

**Comment by robertleeplummerjr** [comment]
Can you throw this into jsfiddle.net?  Also, what browser, and version of
Medium.js are you using?

On Wed, Jan 14, 2015 at 9:01 PM, Simon <notifications@github.com> wrote:

> medium.js:1725 Uncaught TypeError: Cannot read property 'innerText' of
> undefined
>
> Heres my init:
>
> var ed = $("#createBox");
> new Medium({
>     element: ed[0],
>     modifier: 'auto',
>     placeholder: "What's happening?",
>     autofocus: true
>
> )}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
Chrome, version in medium.js says:  * Version: master

**Comment by netfuelnz** [comment]
http://jsfiddle.net/a76kmk9x/

**Comment by robertleeplummerjr** [comment]
Interesting.  I am unable to reproduce locally.  Can you give me
instructional step by step to reproduce?

On Wed, Jan 14, 2015 at 9:25 PM, Simon <notifications@github.com> wrote:

> http://jsfiddle.net/a76kmk9x/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70030542>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
Yea the fiddle seems to work fine too :( Typical - I have put exactly what I have in my code, into fiddle.

**Comment by robertleeplummerjr** [comment]
I'm intrigued.  Care for a remote session?

On Wed, Jan 14, 2015 at 9:35 PM, Simon <notifications@github.com> wrote:

> Yea the fiddle seems to work fine too :( Typical - I have put exactly what
> I have in my code, into fiddle.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70031256>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
Sure how?

**Comment by robertleeplummerjr** [comment]
Teamviewer?  If you'd like to switch to business/private email:
robertleeplummerjr at gmail or rplummer at visop-dev.com

On Wed, Jan 14, 2015 at 9:37 PM, Simon <notifications@github.com> wrote:

> Sure how?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70031385>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
TM: m24-679-003

**Comment by netfuelnz** [comment]
Skyped u an invite.

**Comment by robertleeplummerjr** [comment]
Linux didn't like it, switched to virtualbox, and it was over before it had
begun.

On Wed, Jan 14, 2015 at 9:39 PM, Simon <notifications@github.com> wrote:

> TM: m24-679-003
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70031529>.
>



-- 
Robert Plummer

**Comment by robertleeplummerjr** [comment]
RobertLeePlummerJr, no notifications as of yet.

On Wed, Jan 14, 2015 at 9:44 PM, Simon <notifications@github.com> wrote:

> Skyped u an invite.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70031928>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
DIdnt like teamviewer or skype?

**Comment by robertleeplummerjr** [comment]
I'm awaiting on skype.

On Wed, Jan 14, 2015 at 9:50 PM, Simon <notifications@github.com> wrote:

> DIdnt like teamviewer or skype?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70032319>.
>



-- 
Robert Plummer

**Comment by netfuelnz** [comment]
Ok - add me "SmartySimon" - NZ

**Comment by robertleeplummerjr** [comment]
https://www.youtube.com/watch?v=hmqyc1VVw4M

On Wed, Jan 14, 2015 at 9:50 PM, Robert Plummer <
robertleeplummerjr@gmail.com> wrote:

> I'm awaiting on skype.
>
> On Wed, Jan 14, 2015 at 9:50 PM, Simon <notifications@github.com> wrote:
>
>> DIdnt like teamviewer or skype?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jakiestfu/Medium.js/issues/130#issuecomment-70032319>
>> .
>>
>
>
>
> --
> Robert Plummer
>



-- 
Robert Plummer

**Comment by thibauds** [comment]
Hi, I think I've the same issue.
with the following init code everything is fine: 
```
new Medium({
	    element: document.getElementById('editor'),
	    mode: Medium.partialMode,
	    placeholder: 'Your comment'
});
```
but if I try to use the maxLength option with the following init code:
```
new Medium({
	    element: document.getElementById('editor'),
	    mode: Medium.partialMode,
	    placeholder: 'Your comment',
	    maxLength:2000
});
```
I got the same error message:
```
Uncaught TypeError: Cannot read property 'innerText' of undefined medium.js:1725
Medium.Utilities.utils.text medium.js:1725
intercept.down medium.js:195
```

I installed Medium.js using ```bower install medium.js``` and it seems to install an old version.

**Comment by meshenka** [comment]
Same issue different code

```javascript
   $(function() {
        $('*[data-update="patch"]').each(function(index, value) {
            var $value = $(value);
            $value.on('dblclick', function(){

                new Medium({
                    element: value,
                    mode: M.inlineMode,
                    maxLength: 25,
                    placeholder: 'edit'
                });
                return false;
            });
        });
    });
```
Basicaly i got a table i want the editable behavior when double clicking a cell
i got the `Uncaught TypeError: Cannot read property 'innerText' of undefined` as soon as i change the value of the cell

**Comment by pheara** [comment]
Same here. Removing maxLength from the config, fixed the errors and also an issue where could still enter newlines while in `inlineMode` by pressing enter twice, with weird cursor-jump in between (which reads  a bit like #189).

Using Chrome 51 on Windows 7. My config:

``` javascript
// initialising editor. see http://jakiestfu.github.io/Medium.js/docs/
this.medium = new Medium({
  element: this.textField().querySelector('.medium-mount'),
  modifier: 'auto',
  placeholder: this.placeholder,
  autoHR: false, //if true, inserts <hr> after two empty lines
  mode: Medium.inlineMode, // no newlines, no styling
  //maxLength: this.maxChars, 
  tags: {   },
  attributes: {
    remove: ['style', 'class'] 
  },
});
this.medium.placeholder.style = ""; //remove the inline-styles placed by medium.js
```

**Comment by vhmth** [comment]
Yep removing `maxLength` fixes it. I tried using `data-medium-maxLength`, but that seems to get set to `data-medium-maxlength` when rendered from an underscore template (which I'm assuming causes `Medium.js` to not pick up on it).
